enterprise/trackedusers/users_reporter.go

```diff
@@ -0,0 +1,223 @@
+package trackedusers
+
+import (
+	"context"
+	"database/sql"
+	"encoding/hex"
+	"fmt"
+	"time"
+
+	"github.com/rudderlabs/rudder-server/jobsdb"
+	migrator "github.com/rudderlabs/rudder-server/services/sql-migrator"
+	txn "github.com/rudderlabs/rudder-server/utils/tx"
+
+	"github.com/rudderlabs/rudder-go-kit/config"
+	"github.com/rudderlabs/rudder-go-kit/logger"
+	obskit "github.com/rudderlabs/rudder-observability-kit/go/labels"
+
+	"github.com/lib/pq"
+	"github.com/samber/lo"
+	"github.com/segmentio/go-hll"
+	"github.com/spaolacci/murmur3"
+	"github.com/tidwall/gjson"
+)
+
+const (
+	idTypeUserID                       = "userID"
+	idTypeAnonymousID                  = "anonymousID"
+	idTypeUserIDAnonymousIDCombination = "userIDAnonymousIDCombination"
+
+	// changing this will be non backwards compatible
+	murmurSeed = 123
+)
+
+type UsersReport struct {
+	WorkspaceID              string
+	SourceID                 string
+	UserIDHll                *hll.Hll
```

Comment:
[minor] for consistancy
```suggestion
	UserIDHll                *hll.HLL
```